content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md

```diff
@@ -0,0 +1,169 @@
+# Introduction to Kernel Methods in Machine Learning
+
+### Prerequisites
+- The Reader should have basic knowledge in machine learning [Click here](https://www.digitalocean.com/community/tutorials/an-introduction-to-machine-learning) to read more about machine learning basics.
+- The reader should be able to understand basic terms like Support Vector Machine, classifications, unsupervised learning and supervised learning.
+- Before digging deeper into the operation of Kernel Methods, it is necessary to first understand support vector machines, or SVMs, because kernels are implemented in SVM models.
+
+
+## Introduction
+The concept of "data drawn from some probability distribution" underpins many techniques in statistics and machine learning. A probability distribution is a model for the frequency of data points, i.e., how likely it is to see a particular value, outcome, or other occurrence. The parameters of the underlying distribution are used to define parametric methods (or algorithms).
+
+The assumed model for the likelihood, however, may not be correct; the "actual" distribution may be skewed, multimodal, with different components, and so on. In general, the scientist analyzing the data is unaware of this. This article aims to demonstrate the fundamental concepts underlying kernels as well as their applications in statistics and machine learning.

```

Comment:
reword to "However, the assumed model for the likelihood may not be correct; the "actual" distribution may be skewed, multimodal, with different components, and so on. In general, the scientist analyzing the data is unaware of this. This article aims to demonstrate the fundamental concepts underlying kernels and their applications in statistics and machine learning."